src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -79,11 +79,13 @@ private void InitModelCurve(Autodesk.Revit.DB.Curve crv, bool makeReferenceCurve
             if (mc != null)
             {
                 InternalSetCurveElement(mc);
-                if (!InternalSetSketchPlaneFromCurve(crv))
```

Comment:
hmmm, I think I see what you are saying here - atleast we should change the comment on this `InternalSetSketchPlaneFromCurve` method.